src/main/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporter.java

```diff
@@ -18,30 +17,51 @@
 
     private final static String USER_EVENT_PREFIX = "keycloak_user_event_";
     private final static String ADMIN_EVENT_PREFIX = "keycloak_admin_event_";
-    private static final String PROVIDER_KEYCLOAK_OPENID = "keycloak";
+    private final static String PROVIDER_KEYCLOAK_OPENID = "keycloak";
+
     private final static PrometheusExporter INSTANCE = new PrometheusExporter();
-    private final static CollectorRegistry registry = CollectorRegistry.defaultRegistry;
-    private final static Map<String, Counter> counters = new HashMap<>();
 
-    private final static Counter totalLogins = Counter.build()
+    // package private by on purpose
```

Comment:
I don't think this comments are required, perhaps one time only, preceding all package private declarations.